frontend/test/__support__/e2e/test_tables.js

```diff
@@ -54,3 +56,48 @@ export const scoreboard_actions = async dbClient => {
 
   return true;
 };
+
+export const many_data_types = async dbClient => {
+  const tableName = "many_data_types";
+
+  await dbClient.schema.dropTableIfExists(tableName);
+
+  await dbClient.schema.createTable(tableName, table => {
+    table.increments("id").primary();
+    table.uuid("uuid");
+
+    table.integer("integer");
+    table.integer("integerUnsigned").unsigned();
+    table.tinyint("tinyint");
+    table.tinyint("tinyint1", 1);
+    table.smallint("smallint");
+    table.mediumint("mediumint");
+    table.bigInteger("bigint");
+
+    table.string("string");
+    table.text("text");
+
+    table.float("float");
+    table.double("double");
+    table.decimal("decimal");
+
+    table.boolean("boolean");
+
+    table.date("date");
+    table.dateTime("datetime");
+    table.dateTime("datetimeTZ", { useTz: true });
+    table.time("time");
+    table.timestamp("timestamp");
+    table.timestamp("timestampTZ", { useTz: true });
+
+    table.json("json");
+    table.jsonb("jsonb");
+
+    table.enu("enum", ["alpha", "beta", "gamma", "delta"]);
+
+    table.binary("binary");
+  });
+
+  await dbClient(tableName).insert(many_data_types_data);
+  return true;
```

Comment:
I've probably missed this, but curios why do we have to `return true` here?

Reply:
Cypress task needs to return something.

Reply:
From the documentation: https://docs.cypress.io/api/commands/task#Usage

> If you do not need to return a value, explicitly return null to signal that the given event has been handled.